Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml

```diff
@@ -1,7 +1,7 @@
 # Used by the Vulcan, L6SAW, Mark 1, Ceremonial, and the AKMS. Unsure if there's more.
 - type: entity
   id: BulletLightRifle
-  name: bullet (.20 rifle)
+  name: bullet (.30 rifle)
```

Comment:
.30 rounds are real!

Reply:
i have many things to say about the fact that lightrifles shoot .30 but that's a conversation for a different day